lang/cs/Org.Apache.REEF.Tests/Functional/IMRU/IMRUBroadcastReduceWithFilePartitionDataSetTest.cs

```diff
@@ -0,0 +1,360 @@
+                var tmpFileFodler = Directory.GetCurrentDirectory() + tempFile;
+                Assert.True(Directory.Exists(tmpFileFodler));
+
+                int count = 0;
+                var e = (IEnumerable<Row>)partition.GetPartitionHandle();
+
+                foreach (var row in e)
+                {
+                    Logger.Log(Level.Info, "Data read {0}: ", row.GetValue());
+                    count++;
+                }
+
+                Logger.Log(Level.Info, "TestSenderMapFunction: TaskId: {0}, count {1}", taskId.Length, count);
+                Assert.Equal(count, DataCount);
+            }
+
+            /// <summary>
+            /// Simple test Map function
+            /// </summary>
+            /// <param name="mapInput">integer array</param>
+            /// <returns>The same integer array</returns>
+            int[] IMapFunction<int[], int[]>.Map(int[] mapInput)
+            {
+                _iterations++;
+
+                Logger.Log(Level.Info, "Received value {0} in iteration {1}.", mapInput[0], _iterations);
+
+                if (mapInput[0] != _iterations)
+                {
+                    Exceptions.Throw(new Exception("Expected value in mappers different from actual value"), Logger);
+                }
+
+                return mapInput;
+            }
+        }
+
+        /// <summary>
+        /// This method builds partition dataset configuration. It uses local file system and test RowSerializer
+        /// </summary>
+        /// <param name="numberofMappers"></param>
+        /// <returns></returns>
+        protected override IConfiguration BuildPartitionedDatasetConfiguration(int numberofMappers)
+        {
+            const string tempFileName = "REEF.TestLocalFileSystem.tmp";
+            string sourceFilePath = Path.Combine(Path.GetTempPath(), tempFileName);
+
+            var cm = FileSystemInputPartitionConfiguration<IEnumerable<Row>>.ConfigurationModule;
+
+            for (var i = 0; i < numberofMappers; i++)
+            {
+                MakeLocalTestFile(sourceFilePath + i, CreateTestData(DataCount));
+                cm = cm.Set(FileSystemInputPartitionConfiguration<IEnumerable<Row>>.FilePathForPartitions, sourceFilePath + i);
+            }
+            
+            return cm.Set(FileSystemInputPartitionConfiguration<IEnumerable<Row>>.CopyToLocal, "true")
+                     .Set(FileSystemInputPartitionConfiguration<IEnumerable<Row>>.FileSerializerConfig, GetRowSerializerConfigString())
+                     .Build();
+        }
+
+        /// <summary>
+        /// Create test data as bytes
+        /// </summary>
+        /// <param name="dataNumber"></param>
+        /// <returns></returns>
+        private static byte[] CreateTestData(int dataNumber)
+        {
+            var bytes = new byte[dataNumber];
+            for (int i = 0; i < dataNumber; i++)
+            {
+                bytes[i] = (byte)i;
```

Comment:
A little readability improvement: if we write `(byte)(i + 'a')` here, test data generated will be readable characters. I think it's better to work with human-readable data if possible